src/EditorFeatures/VisualBasicTest/Completion/CompletionProviders/CompletionListTagCompletionProviderTests.vb

```diff
@@ -342,6 +342,32 @@ End Class
             VerifyItemIsAbsent(markup, "e As E")
         End Sub
 
+        <WorkItem(3518, "https://github.com/dotnet/roslyn/issues/3518")>
+        <Fact, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Sub NotInTrivia()
+            Dim markup = <Text><![CDATA[
+Class C
+    Sub Test()
+        M(Type2.A)
+        ' $$
```

Comment:
I'm confused - why is the provider kicking in here at all (after the `)`)?  Not saying that this fix is the wrong thing to do, but it seems like (and testing confirms) we will still have problems with pre-selection happening in contexts like this, where it shouldn't even if it's not a comment.